.github/workflows/simulation-1.2.yml

```
@@ -111,6 +110,7 @@ jobs:
       THREAD_VERSION: 1.2
       MAC_FILTER: 1
       INTER_OP: 1
+      BBR: 0
```

```suggestion
      INTER_OP_BBR: 0
```
Let's be clear is for turning on BBR inter op?

---

updated.